pyproject.toml

```
@@ -99,6 +99,9 @@ disallow_untyped_defs = false
 addopts = [
   # `pytest-xdist`:
   "--numprocesses=auto",
+
+  # This distribution method is useful if the run times of different tests vary greatly,
```

Code comments that are not on the same line tend to drift away from the lines they refer to over time. Referencing the exact context may help notice this:
```suggestion
  # The `worksteal` distribution method is useful if the run times of different tests vary greatly,
```

---

Thanks, applied the fix.